edge/src/service/github/github.service.ts

```diff
@@ -32,6 +34,40 @@ class GithubService extends HttpClient {
       path: `/orgs/${this.organizationName}/members`,
     });
   }
+
+  async getOrgRepos(): Promise<[GithubRepositoryModel]> {
+    return await this.call<[GithubRepositoryModel]>({
+      method: HttpMethod.GET,
+      path: `/orgs/${this.organizationName}/repos`,
+    });
+  }
+
+  async getOrgContributors(): Promise<GithubContributorModel[]> {
+    const repos = await this.getOrgRepos();
+    const promises = repos.map((repo) => {
+      return this.call<GithubContributorModel[]>({
+        method: HttpMethod.GET,
+        path: `/repos/${this.organizationName}/${repo.name}/contributors`,
```

Comment:
with great respect, please explain how `this.organizationName` and `repo.name` get passed in here ? The function don't have inline comment or may be it's a JS style ? so i can't understand it.

Thank you.

Reply:
I think it's a common javascript feature that used nowadays, so I will explain it to you here instead.

it called string interpolation, you passing variable inside a string so that HTTP call invoked inside a [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) then it will create [promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) of contributor of every repos ex ` '/repos/probolinggo-dev/monosource/contributors' ` then the promises will be resolved using [Promise All](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) that return a bunch of Contributor in form of two-dimensional array GithubContributorModel[][]

![image](https://user-images.githubusercontent.com/10051243/86879265-eabd4280-c114-11ea-90b7-d3b7202ed245.png)

Reply:
Thank you for replying and explain my question clearly with reference included. I will check it out 👌